spec/system/admin/activity_spec.rb

```diff
@@ -373,4 +373,24 @@
       end
     end
   end
+
+  scenario "Shows moderation activity without html tags in description" do
+    proposal = create(:proposal, title: "Make Pluto a planet again",
+                                 description: "<p>Description with html tag</p>")
+
+    visit proposal_path(proposal)
```

Comment:
Since this test is almost identical to an existing one, what do you think about merging them together? :thinking: That way we'd make the tests faster.

If you'd rather keep the tests separate, then how about removing the part with the first `vistit` and clicking on "Hide"? That part is already tested in the other test, and so in this one we could setup the database and visit the `admin_activity_path` directly, saving a couple of requests.